cookbook/bundles/best_practices.rst

```diff
@@ -115,22 +115,23 @@ them under the ``cache/`` or ``log/`` directory of the host application. Tools
 can generate files in the bundle directory structure, but only if the generated
 files are going to be part of the repository.
 
-The following classes and files have specific emplacements:
-
-===============================  =============================
-Type                             Directory
-===============================  =============================
-Commands                         ``Command/``
-Controllers                      ``Controller/``
-Service Container Extensions     ``DependencyInjection/``
-Event Listeners                  ``EventListener/``
-Model classes [1]                ``Model/``
-Configuration                    ``Resources/config/``
-Web Resources (CSS, JS, images)  ``Resources/public/``
-Translation files                ``Resources/translations/``
-Templates                        ``Resources/views/``
-Unit and Functional Tests        ``Tests/``
-===============================  =============================
+The following classes and files have specific emplacements (some are mandatory
+and others are just conventions followed by most developers):
+
+===============================  =============================  ================
+Type                             Directory                      Mandatory?
+===============================  =============================  ================
+Commands                         ``Command/``                   Yes
```

Comment:
I don't think it's that easy. For example: how do you put the translations in `<your-bundle>/translations/*` instead of `<your-bundle>/Resources/translations/` or the templates in `<your-bundle>/templates/*` instead of `<your-bundle>/Resources/views/*` ?